frontend/templates/troubleshooting/Resource.tsx

```diff
@@ -134,20 +133,26 @@ function ResourceBox({
    ...props
 }: React.PropsWithChildren<StackProps>) {
    return (
-      <Stack
-         alignSelf="stretch"
-         justify="flex-start"
-         align="flex-start"
-         spacing="0px"
-         overflow="hidden"
+      <LinkBox
          borderColor="gray.400"
          borderWidth="1px"
-         borderRadius="4px"
+         borderRadius="md"
          minHeight="88px"
+         padding={3}
+         display="flex"
+         width="100%"
+         transition={`border-color var(--chakra-transition-duration-normal)`}
+         sx={{
+            '@media (hover: hover)': {
+               _hover: {
+                  borderColor: 'brand.500',
+               },
+            },
```

Comment:
Why the @media query? I think you can use _hover={{borderColor: 'brand.500'}}

Reply:
Maybe we should to simplify... I prefer this pattern because it only applies the styles when a mouse is used.

Reply:
> because it only applies the styles when a mouse is used

Woah, that's a cool trick / something I was not tracking. I'm fine leaving it as is. I'd almost open an issue for them to extend the [psudeo](https://chakra-ui.com/docs/styled-system/style-props#pseudo) style props. They have `_mediaDark` and `_mediaReduceMotion` already establishing a pattern.

Reply:
I'll standardize here and use the built-in. My bad, I should have done that from the start.

Reply:
https://github.com/iFixit/react-commerce/pull/1816/commits/170549ba71521b796eb197cc8f47e5b41e52e0b8